appengine/flexible/RedisCache/README.md

```diff
@@ -29,26 +29,29 @@ when running in Google App Engine Flexible Environment.
 
 ## Run Locally
 
-### Using PowerShell
+### ![PowerShell](../.resources/powershell.png)Using PowerShell
 ```psm1
 PS > dotnet restore
 PS > dotnet run
 ```
 
-### Using Visual Studio
+### ![Visual Studio](../.resources/visual-studio.png)Using Visual Studio
+
```

Comment:
1.  In Solution Explorer, right-click the **RedisCache** project and choose **Set as StartUp Project**
2.  Press F5.